cspell.speaker-names.txt

```diff
@@ -1,13 +1,15 @@
-Rizèl
-Scarlett
-Nnenna
-Mitropoulos
 Abhijit
 Ajit
+Blass
+Dimitri
+Farolino
+Kamath
 Mariah
+Mitropoulos
 Murg
-Kamath
-Farolino
-Dimitri
 Ndukwe
-Blass
\ No newline at end of file
+Nnenna
+Pariss
+Puskarevic
+Rizèl
+Scarlett
```

Comment:
I'm quite shocked I didn't keep those in alphabetical order, thanks for doing that!

Reply:
Oh I didn't do this manually, I think it happened automatically